[FIX]l10n_es_aeat_mod303: exportación a BOE 2021

jesusVMayor opened the pull request:
En v11 y v12 el campo apply_sign está con default a True, el valor de "Identificación: Sujeto pasivo con volumen anual de operaciones distinto de cero (art. 121 LIVA)" se debe de exportar sin signo, si no lo pone siempre a 0

pedrobaeza commented:
Anda, justo estaba yo reparando esto en versiones superiores. Es eso y del swift que no se debe rellenar por defecto.

pedrobaeza committed:
[FIX] l10n_es_aeat_mod303: Last BOE export inaccuracies

- Don't apply sign to "Volumen operaciones"
- Don't export SWIFT if Spanish or European bank

pedrobaeza commented:
Mira mi commit para que veas los dos cambios hechos: a7f5a6931021bf4fdecdff70257a846417b5efc0

jesusVMayor commented:
ah, que ni en el fichero correcto lo había hecho :man_facepalming: el cambio es lo mismo así que hago cherry-pick

pedrobaeza commented:
/ocabot merge nobump

HaraldPanten commented:
¿Pedro, lo pasas a las siguientes versiones?

Gracias.

pedrobaeza commented:
Te has perdido una issue por ahí :wink:

HaraldPanten commented:
Ya veo, ya... 👍
